rclcpp_action/include/rclcpp_action/server.hpp

```diff
@@ -345,7 +346,8 @@ class Server : public ServerBase, public std::enable_shared_from_this<Server<Act
 
   /// Signature of a callback that accepts or rejects goal requests.
   using GoalCallback = std::function<GoalResponse(
-        const GoalUUID &, std::shared_ptr<const typename ActionT::Goal>)>;
+        const GoalUUID &,
+        std::shared_ptr<const typename ActionT::Goal>)>;
```

Comment:
This change (and the one below) seem to be no-ops, just reformatting code.  I'm going to request that you remove these changes, in which case it is much clearer that this PR does not change API or ABI.